mitiq/calibration/calibrator.py

```diff
@@ -49,10 +50,26 @@ def add_result(
         ideal_val: float,
         noisy_val: float,
         mitigated_val: float,
+        log: bool = False,
     ) -> None:
         self.mitigated[strategy.id, problem.id] = mitigated_val
         self.noisy[strategy.id, problem.id] = noisy_val
         self.ideal[strategy.id, problem.id] = ideal_val
+        if not log:
+            return
+        mitigated_better = abs(ideal_val - mitigated_val) < abs(
+            ideal_val - noisy_val
+        )
+        performance = "✅" if mitigated_better else "❌"
+        print(
+            f"Running {problem.type} circuit using:",
```

Comment:
Technically we are not "running" the circuit. We completed the execution and we are showing the result.

Reply:
Technically true, but this code would only ever run _just_ after executing a circuit. Do you think a word change is in order, or would you like the logging code moved to the `run` function?

Reply:
The current message can be confusing in some cases: e.g. when there is a long queue in the hardware or when a circuit execution fails.

Both solutions you suggested would work for me.

Reply:
Reworded in [2af1b88](https://github.com/unitaryfund/mitiq/pull/1714/commits/2af1b88a05610ce387af63fe6cff16f61d7f9e2c).